### PR TITLE
Adding a test for kubemci remove-clusters

### PR DESCRIPTION
**Pull request by nikhiljindal**
Ref https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/58

Adding an e2e test for `kubemci remove-clusters` command. 
The test creates an ingress and then removes it from all clusters.

cc @G-Harmon

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/network/ingress.go [hunk 1]
```diff
@@ -636,6 +636,22 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 		It("should create ingress with backend HTTPS", func() {
 			executeBacksideBacksideHTTPSTest(f, jig, ipName)
 		})
+
+		It("should remove clusters as expected", func() {
+			jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "http"), ns, map[string]string{
```
[the remark this answers is not in the document]
Comment (nikhiljindal) [reply test/e2e/network/ingress.go]: Done

File: test/e2e/network/ingress.go [hunk 2]
```diff
@@ -636,6 +636,22 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
...
+				framework.IngressStaticIPKey: ipName,
+			}, map[string]string{})
+			jig.WaitForIngress(false /*waitForNodePort*/)
+			name := jig.Ingress.Name
+			// Verify that the ingress is spread to 1 cluster as expected.
+			verifyKubemciStatusHas(name, "is spread across 1 cluster")
+			// Reuse the ingress file created while creating the ingress.
+			filePath := framework.TestContext.OutputDir + "/mci.yaml"
```
[the remark this answers is not in the document]
Comment (nikhiljindal) [reply test/e2e/network/ingress.go]: Done

File: test/e2e/network/ingress.go [hunk 3]
```diff
@@ -690,6 +706,17 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 	})
 })
 
+// verifyKubemciStatusHas fails if kubemci get-status output for the given mci does not have the given statusSubStr.
+func verifyKubemciStatusHas(name, statusSubStr string) {
```
[the remark this answers is not in the document]
Comment (nikhiljindal) [reply test/e2e/network/ingress.go]: renamed to `expectedSubStr`

**Comment by nikhiljindal** [comment]
Thanks for the review @madhusudancs 
Updated as per comments

**Review by madhusudancs — APPROVED** [review]
/lgtm
/approve

**Comment by nikhiljindal** [comment]
Did a trivial rebase.

Lets see if I can self lgtm.

/lgtm

**Pull request merged**